example/ios/Podfile

```diff
@@ -17,6 +17,8 @@ end
 target 'InbrainReactNativeExample' do
   config = use_native_modules!
 
+  pod 'InBrainSurveys', :git => 'https://github.com/inbrainai/inBrainSurveys_SDK.git', :tag => '2.5.0'
```

Comment:
Hmmm... that shouldn't be added. It was just for testing purposes during development

Reply:
Done